Dockerfile

```diff
@@ -3,11 +3,21 @@ FROM openjdk:11
 LABEL softartdev <artik222012@gmail.com>
 
 ENV ANDROID_SDK_URL https://dl.google.com/android/repository/commandlinetools-linux-7583922_latest.zip
-ENV ANDROID_API_LEVEL android-30
-ENV ANDROID_BUILD_TOOLS_VERSION 30.0.3
+ENV ANDROID_API_LEVEL android-33
+ENV ANDROID_BUILD_TOOLS_VERSION 33.0.2
 ENV ANDROID_HOME /usr/local/android-sdk-linux
 ENV ANDROID_VERSION 30
 ENV PATH $PATH:$ANDROID_HOME/tools:$ANDROID_HOME/tools/bin:$ANDROID_HOME/platform-tools:$ANDROID_HOME/cmdline-tools/bin
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update
+RUN apt-get install -y curl gnupg unzip
+RUN curl -s https://apt.corretto.aws/corretto.key | apt-key add -
+RUN echo 'deb https://apt.corretto.aws stable main' | tee /etc/apt/sources.list.d/corretto.list
+RUN apt-get update
+RUN apt-get install -y java-17-amazon-corretto-jdk
+ENV JAVA_HOME=/usr/lib/jvm/java-17-amazon-corretto/
```

Comment:
There is no need to download another version of Java. There is an easier way, just update the first line of this file to `FROM openjdk:17`. But thanks for this pull request, I'm going to update it soon.

Reply:
`unzip` is included in debian, which is not used in default `openjdk` images since version 12. But [bullseye or buster](https://github.com/docker-library/docs/tree/master/openjdk#image-variants) version tags still use debian, so `openjdk:17-bullseye` is appropriate.

Reply:
I have [updated the image](https://hub.docker.com/layers/softartdev/android-fastlane/33/images/sha256-f9dcb807d7062c1113d21df527b29f7e506203fd02032ea93fce5550266f1975?context=repo), you can check it out.